vnext/src/Libraries/Components/Picker/PickerWindows.tsx

```diff
@@ -85,20 +85,19 @@ export class Picker extends React.Component<IPickerProps, State> {
   }
 
   public render(): JSX.Element {
-    const props = {...this.props};
-    props.onStartShouldSetResponder = () => true;
-    props.onResponderTerminationRequest = () => false;
-    props.style = this.props.style;
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
There's a setting to disable this check when adjacent to the rest parameter that's off by default, but not in stock RN eslint rules that we're using apparently. I'd be fine changing that as well.